libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -880,6 +882,8 @@ singleton AC_Fence depends AP_FENCE_ENABLED
 singleton AC_Fence rename fence
 singleton AC_Fence method get_breaches uint8_t
 singleton AC_Fence method get_breach_time uint32_t
+singleton AC_Fence method get_safe_alt_min float
+singleton AC_Fence method get_safe_alt_max float
 
```

Comment:
Sorry I have to say again, AC_Fence present() should be exposed as a binding. IMO Expecting scripts to correctly duplicate the logic that should be encapsulated in cpp is not a good idea. For example: now any scripts that have built their own fence present() logic will now have to be updated/changed to correctly interpret the new configured logic.

Reply:
Ok added